### PR TITLE
Test results directive --without-syschar and --thin-results

### PR DESCRIPTION
**Pull request by yuumasato**

### CODE REVIEW

**Comment by mpreisler** [comment]
@openscap-jenkins test this please

**Comment by mpreisler** [comment]
I'd expect `Makefile.am` changes but I don't see them in the diff.

File: tests/nist/nist_test.sh [hunk 1]
```diff
@@ -21,6 +21,60 @@ function test_nist {
...
+function test_results_directive {
+	test_dir="$1"
+	run_script="$(cd "${builddir}"; pwd)/run"
+	# make sure the output dir in builddir exists
+	mkdir -p "${builddir}/tests/nist/$test_dir/"
+	local datastream="ind_family_test-datastream.xml"
+
+	local result="${builddir}/tests/nist/${test_dir}/results_directive.results_arf.xml"
+	# the tests of this evaluation dont need to pass, return 0
+	${run_script} $OSCAP xccdf eval --profile "xccdf_gov.nist_profile_family-validation" --results-arf ${result} "${srcdir}/$test_dir/${datastream}" > /dev/null && 0
```
Comment (dahaic) [inline tests/nist/nist_test.sh]: Not sure what the construct `&& 0` means - should be `|| true`?

Comment (mpreisler) [inline tests/nist/nist_test.sh]: Good catch, this does look fishy.

Comment (yuumasato) [inline tests/nist/nist_test.sh]: Yeah, I was a little bit confused by exit statuses.
But I still don't know why it is working with `&&` if the evaluation of this data stream returns 2.

File: tests/nist/nist_test.sh [hunk 2]
```diff
@@ -21,6 +21,65 @@ function test_nist {
 	fi
 }
 
+function test_results_directive {
+	test_dir="$1"
+	run_script="$(cd "${builddir}"; pwd)/run"
```
Comment (dahaic) [inline tests/nist/nist_test.sh]: As we have test_common.sh sourced in this file, you can use $OSCAP, instead of defining run_script by yourself.

Comment (yuumasato) [inline tests/nist/nist_test.sh]: Yes, I noticed that, but wanted to keep it consistent with `test_nist()`, which also uses `${run_script}`.

Comment (mpreisler) [inline tests/nist/nist_test.sh]: Looks like a mistake in the original PR to me. Let's use `$OSCAP` everywhere.

**Comment by mpreisler** [comment]
Looks good to me now, ACK.

**Pull request merged**